src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferConstCharOverConstUnitStringTests.cs

```diff
@@ -493,11 +493,12 @@ private void TestMethod(int value)
         { 
             StringBuilder sb = new StringBuilder();
             const string ch = ""a"";
-            sb.Append($""{ch}"");
+            sb.Append([|$""{ch}""|]);
         } 
     } 
 }";
-            await VerifyCS.VerifyAnalyzerAsync(interpolatedString_cs);
+
+            await VerifyCS.VerifyCodeFixAsync(interpolatedString_cs, interpolatedString_cs);
```

Comment:
@mavasani No fix is offered for this edge case. Is this behavior okay or should I file an issue?

Reply:
I think it is fine.